bizhawk-crowd-shuffler.lua

```diff
@@ -62,12 +63,14 @@ end
 
 local function main()
    -- purge socket data
+   comm.socketServerSetTimeout(12)
    comm.socketServerResponse()
 
    while true do -- The main cycle that causes the emulator to advance and trigger a game switch.
         frame = frame + 1
 
         if (frame % frame_check_mod) == 0 then
+            frame = 0
```

Comment:
I'm curious about this line. What is the intention of the change?

Reply:
Just me being paranoid about overflows when leaving the variable unattended for too long. Might not be necessary unless you plan to run it for years.

Reply:
Ah I see. I have actually seen that in other projects with Lua. Thanks for taking a moment to explain ✅